sub-pages/_single.html

```diff
@@ -85,20 +84,43 @@ <h2 class="h3">
                     block
                     block__padded-top
                     block__border-top">
-        {{ sub_page.body_content }}
+        {% set content = { 'markup' : sub_page.body_content } %}
+        {% for form_name in vars.forms %}
```

Comment:
This isn't a great solution. Need to rethink how we handle the forms and the form placeholders coming from WP.